storey/flow.py

```diff
@@ -1464,6 +1502,13 @@ class ParallelExecutionRunnable:
         block the main event loop thread.
     * "asyncio" – To run in an asyncio task. This is appropriate for I/O tasks that use asyncio, allowing the event
         loop to continue running while waiting for a response.
+    * "shared_executor" – Reuses an external executor (typically managed by the flow or context)
```

Comment:
I think it's important to explain that this is specifically for sharing the model between different steps in the graph. I.e. there is never a reason to use it if there is only one model runner in the graph.